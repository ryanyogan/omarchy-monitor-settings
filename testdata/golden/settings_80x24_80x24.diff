Visual regression detected!

=== EXPECTED ===
  [90m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m│[0m                            [1;34mDisplay Settings[0m                            [90m│[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                
  [35m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [1;35m⚙️ Application Settings[0m                                               [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [1;34m📱 Application Info[0m                                                   [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [37m  Version: [32m1.0.0[0m[0m                                                      [35m│[0m    
  [35m│[0m  [37m  Theme: [35mTerminal Adaptive (TrueColor (24-bit), Light)[0m[0m                [35m│[0m    
  [35m│[0m  [37m  Mode: [32mLive[0m[0m                                                          [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [1;36m🔍 Detection Methods[0m                                                  [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [37m  Hyprctl: [31m✗ Not found[0m[0m                                                [35m│[0m    
  [35m│[0m  [37m  wlr-randr: [31m✗ Not found[0m[0m                                              [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [1;33m⚙️ Configuration[0m                                                      [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [37m  Target: [32mHyprland + Wayland[0m[0m                                          [35m│[0m    
  [35m│[0m  [37m  Fallbacks: [34mwlr-randr[0m[0m                                                [35m│[0m    
  [35m│[0m  [37m  Font Scaling: [35mGTK, Alacritty, Neovim[0m[0m                                [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [3;90m💡 Press Esc to return to the main menu[0m                               [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m╰────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                
  [90m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m│[0m       [1;32m↑↓[0m  [37mnavigate[0m    [1;34m⏎[0m  [37mselect[0m    [1;33mh[0m  [37mhelp[0m    [1;35mesc[0m  [37mback[0m    [1;31mq[0m  [37mquit[0m     [90m│[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────╯[0m    

=== ACTUAL ===
  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │                            Display Settings                            │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    
                                                                                
  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │                                                                        │    
  │  ⚙️ Application Settings                                               │    
  │                                                                        │    
  │  📱 Application Info                                                   │    
  │                                                                        │    
  │    Version: 1.0.0                                                      │    
  │    Theme: Terminal Adaptive (Basic, Dark)                              │    
  │    Mode: Live                                                          │    
  │                                                                        │    
  │  🔍 Detection Methods                                                  │    
  │                                                                        │    
  │    Hyprctl: ✗ Not found                                                │    
  │    wlr-randr: ✗ Not found                                              │    
  │                                                                        │    
  │  ⚙️ Configuration                                                      │    
  │                                                                        │    
  │    Target: Hyprland + Wayland                                          │    
  │    Fallbacks: wlr-randr                                                │    
  │    Font Scaling: GTK, Alacritty, Neovim                                │    
  │                                                                        │    
  │  💡 Press Esc to return to the main menu                               │    
  │                                                                        │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    
                                                                                
  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │       ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit     │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    

=== DIFF INFO ===
Expected length: 5013
Actual length: 4109
Expected hash: 4ab731c305184cd9f5c707d639d08189c3de3771b81cd1660ec6656d55d37b61
Actual hash: 1ed593bfdebdf5d02ad57bcb97308b834058c9e539f10abcb65f8939bfefdd8c
