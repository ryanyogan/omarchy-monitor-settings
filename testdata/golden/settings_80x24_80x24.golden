# Visual Golden File
# Name: settings_80x24
# Dimensions: 80x24
# Hash: 4ab731c305184cd9f5c707d639d08189c3de3771b81cd1660ec6656d55d37b61

  [90m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m│[0m                            [1;34mDisplay Settings[0m                            [90m│[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                
  [35m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [1;35m⚙️ Application Settings[0m                                               [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [1;34m📱 Application Info[0m                                                   [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [37m  Version: [32m1.0.0[0m[0m                                                      [35m│[0m    
  [35m│[0m  [37m  Theme: [35mTerminal Adaptive (TrueColor (24-bit), Light)[0m[0m                [35m│[0m    
  [35m│[0m  [37m  Mode: [32mLive[0m[0m                                                          [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [1;36m🔍 Detection Methods[0m                                                  [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [37m  Hyprctl: [31m✗ Not found[0m[0m                                                [35m│[0m    
  [35m│[0m  [37m  wlr-randr: [31m✗ Not found[0m[0m                                              [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [1;33m⚙️ Configuration[0m                                                      [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [37m  Target: [32mHyprland + Wayland[0m[0m                                          [35m│[0m    
  [35m│[0m  [37m  Fallbacks: [34mwlr-randr[0m[0m                                                [35m│[0m    
  [35m│[0m  [37m  Font Scaling: [35mGTK, Alacritty, Neovim[0m[0m                                [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m  [3;90m💡 Press Esc to return to the main menu[0m                               [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m│[0m                                                                        [35m│[0m    
  [35m╰────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                
  [90m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m│[0m       [1;32m↑↓[0m  [37mnavigate[0m    [1;34m⏎[0m  [37mselect[0m    [1;33mh[0m  [37mhelp[0m    [1;35mesc[0m  [37mback[0m    [1;31mq[0m  [37mquit[0m     [90m│[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────╯[0m    